### PR TITLE
Improve the PHA simulation documentation

### PR DESCRIPTION
**Pull request by DougBurke**
# Summary

Fix up some links in the RTD documentation for simulating PHA data.

# Details

This only changes files used to create the RTD site; it does not change the code base.

Some of these changes are possible now that #1268 has landed, the others are fixing references/links (e.g. changing `sherpa.astro.io.fake_pha` to `sherpa.astro.fake.fake_pha`). There's no substantive change to the text.

### CODE REVIEW

**Comment by DougBurke** [comment]
@hamogu - I have two questions for the docs (I can't seem to link to the lines because GitHub has auto-converted it to markdown...

1. In the "Sample background from a PHA file" section you have

```
  >>> data.set_background(read_pha('9774.arf'))
```

This is reading in an ARF. I'd change it to `9774_bg.pi` but given the text in this section I wasn't quite sure this was correct.

2. In this section you say

```
The reason for that is it is often
useful to read in a file with ``data = read_pha('9774_bg.pi')``, which
might automatically read in the background as well.
```

Should this be `9774_bg.pi` or `9774.pi` (as you are unlikely - but not impossible - to have BACKFILE set on a background file)

**Comment by hamogu** [comment]
You are correct on both counts. Those are all my mistakes! I guess I should work on #816 which would have caught those... GH does not allow me to out in a suggestion for the first one because it's more than 5 lines from any code you touched already, so please go and change them as you suggested. Thanks!

**Comment by DougBurke** [comment]
Hopefully 2a30c32 is correct.

**Review by hamogu — APPROVED** [review]

File: docs/evaluation/simulate.rst [hunk 1]
```diff
@@ -160,19 +160,19 @@ before, the scale of the background scaling has to be set::
...
 background count number as the expected value and performs a Poisson
 draw. The background drawn from the Poisson distribution is than added
 to the simulated source spectrum (the sum of two Poisson distributions
 is a Poisson distribution again). This works best if the background is
 well exposed and has a large number or counts.
 
-Why do we need to set the ``add_bks=True`` argument and do not simply
+Why do we need to set the ``add_bkgs=True`` argument and do not simply
 use all available backgrounds? The reason for that is it is often
 useful to read in a file with ``data = read_pha('9774_bg.pi')``, which
```
[the remark this answers is not in the document]
Comment (DougBurke) [reply docs/evaluation/simulate.rst]: Didn't I male this change in 2a30c32 ?

[the remark this answers is not in the document]
Comment (hamogu) [reply docs/evaluation/simulate.rst]: I think I had a hanging comment that GH did not post last time but kept for when I used "add review". Sorry about that. The change is done and I approved the PR.

**Pull request merged**